File: c/3-loops/workout/lw-5.c
#include<stdio.h>

void main()
{
    int n;
    if(scanf("%d",&n)!=1)
    {
        printf("Invalid Input\n");
    }
    if(n<0 || n>=10)
    {
        printf("Invalid Input\n");
    }
    int size =2*n-1;
    for(int i=0;i<size;i++)
    {
        for(int j=0;j<size;j++)
        {
            int top=i;
            int left=j;
            int right=size-1-j;
            int bottom= size-1-i;
            int min=top;
            if(left<min)min=left;
            if(right<min)min=right;
            if(bottom<min)min=bottom;
            printf("%d",n-min);
            
        }
        printf("\n");
    }
}

File: python/07-lists/workout/lw-5.py
try:
    a = list(map(int, input().split()))
    i1 = int(input())
    i2 = int(input())

    size = len(a)
    if  (i1 > size and i2 > size):
        print("Invalid input")
    else:
        a[i1], a[i2] = a[i2], a[i1]
        print(*a)
except:
    print("Invalid input")